src/components/Table/components/Navlink.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Navlink.scss';
+
+const NavLink = ({ label, onClick, className }) => {
+  const handleClick = () => {
```

Comment:
You don't need this function you can use onClick in jsx as well. this function just call function from props. Why you don't use directly